account_invoices_batches_mail_attach_existing_attachment/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def _get_batch_sending_methods(self):
+        values = super()._get_batch_sending_methods()
+        email_index = next(
+            (i for i, v in enumerate(values) if v[0] == "email"), len(values)
+        )
+        values.insert(
+            email_index + 1, ("emailattachments", _("e-mail (Attach documents)"))
+        )
```

Comment:
Please, do this with a loop on the list of tuples and add the new option just after finding "email" option